src/map/Map.js

```diff
@@ -776,6 +776,9 @@ L.Map = L.Evented.extend({
 		// @pane markerPane: HTMLElement = 600
 		// Pane for `Icon`s of `Marker`s
 		this.createPane('markerPane');
+		// @pane labelPane: HTMLElement = 700

```

Comment:
Should be [`650` I guess](https://github.com/Leaflet/Leaflet/blob/a1e43ed3bb3032e6d43c76e53161cec8b03e4e6f/dist/leaflet.css#L87)